ichnaea/service/locate.py

```diff
@@ -378,22 +403,35 @@ def locate(self, data):
 
 
 class CellLocationProvider(AbstractCellLocationProvider):
+
     """
     A CellLocationProvider implements a cell location search using
     the Cell and OCID Cell models.
     """
```

Comment:
Docstring outdated, only single model is used. Maybe we could change `models = (...)` to `model = ...` now that we use a single model everywhere.